crates/pessimistic-proof/src/imported_bridge_exit.rs

```diff
@@ -15,19 +16,17 @@ pub struct ImportedBridgeExit {
     /// Need to verify that the destination network matches the current network, and that
     /// the bridge exit is included in an imported LER
     pub bridge_exit: BridgeExit,
-
     /// The Network ID for the sending network. May not be strictly
     /// necessary depending on the final structure of the Imported Local Exit Root
     pub sending_network: NetworkId,
```

Comment:
Can we remove `sending_network` if we include the global index ?

Reply:
Yeah indeed